### ISSUE TITLE
"invalid comparator" on lerna bootstrap

### ISSUE THREAD

**Issue by selfisekai** [getinsomnia/insomnia#5533]
### Expected Behavior

dependencies should be installed by lerna/npm

### Actual Behavior

```
lerna notice cli v4.0.0
lerna notice filter including "insomnia"
lerna notice filter including dependencies
lerna info filter [ 'insomnia' ]
lerna WARN bootstrap Installing local packages that do not match filters from registry
lerna info Bootstrapping 19 packages
lerna info Installing external dependencies
lerna ERR! npm install --no-save exited 1 in 'insomnia'
lerna ERR! npm install --no-save stderr:
npm ERR! Invalid comparator: github:Kong/protobuf.js#master

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/lauren/.npm/_logs/2022-12-15T17_35_00_779Z-debug-0.log
lerna ERR! npm install --no-save exited 1 in 'insomnia'
lerna WARN complete Waiting for 7 child processes to exit. CTRL-C to exit immediately.
```

### Reproduction Steps

```
npm ci --ignore-scripts
npx lerna bootstrap --scope insomnia --include-dependencies
```

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

2022.7.0

### What operating system are you using?

Other Linux

### Operating System Version

Alpine Linux edge

### Installation method

release source code tarball

### Last Known Working Insomnia version

2022.6.0

**Comment by selfisekai** [comment]
this has solved it for me:
```patch
diff --git a/packages/insomnia/package.json b/packages/insomnia/package.json
index 30644af..fdf063b 100644
--- a/packages/insomnia/package.json
+++ b/packages/insomnia/package.json
@@ -214,7 +214,7 @@
   "dev": {
     "dev-server-port": 3334
   },
-  "overrides": {
-    "protobufjs": "github:Kong/protobuf.js#master"
+  "resolutions": {
+    "protobufjs": "https://github.com/Kong/protobuf.js/commit/eca9f0fcfc816460d5fe899c160e1462c2edf49b.tar.gz"
   }
 }
```

I had an idea this might be about npm version on my machine but it's `9.2.0` (latest release, from a week ago)

**Comment by jackkav** [comment]
Thanks for the report @selfisekai fixed in #5534 which will soon be in develop.
Should solve your issue.
